mkdocs/docs/api.md

```diff
@@ -844,6 +844,19 @@ readable_metrics: [
 [6.0989]]
 ```
 
+<!-- prettier-ignore-start -->
```

Comment:
with the new markdown linter (#1118), this might not be necessary anymore

Reply:
Okay, removing it. I was not sure what this did in the first place. Just observed that other snippets of Info or Note have is wrapped in this tag, hence added the same.